src/Api/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Bit.Api.SecretManagerFeatures.Models.Request;
+using Bit.Api.SecretManagerFeatures.Models.Response;
+using Bit.Api.Utilities;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.SecretManagerFeatures.Projects.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Controllers
+{
+    [SecretsManager]
+    public class ProjectsController : Controller
+    {
+        private readonly IProjectRepository _projectRepository;
+        private readonly ICreateProjectCommand _createProjectCommand;
+        private readonly IUpdateProjectCommand _updateProjectCommand;
+
+        public ProjectsController(IProjectRepository projectRepository, ICreateProjectCommand createProjectCommand, IUpdateProjectCommand updateProjectCommand)
+        {
+            _projectRepository = projectRepository;
+            _createProjectCommand = createProjectCommand;
+            _updateProjectCommand = updateProjectCommand;
+        }
+
+        [HttpPost("organizations/{organizationId}/projects")]
+        public async Task<ProjectResponseModel> CreateAsync([FromRoute] Guid organizationId, [FromBody] ProjectCreateRequestModel createRequest)
+        {
+            if (organizationId != createRequest.OrganizationId)
```

Comment:
Originally for secrets I had organization ID in the creation request body and in the route. I was asked to change it, and we probably want to stay consistent with that approach.

Can just take it out of the request body and call `createRequest.ToProject(ogranizationId)`

It will also eliminate the need for this request validation.